README.md

```diff
@@ -116,11 +116,11 @@ We also include a wrapper for Pact-JS V3.
 
 If you have thoughts or feedback about the DSL, please let us know via slack or open issue.
 
-Currently, only a default for the pact directory is provided by the jest-pact wrapper `jest-pact/v3`.
+Currently, only a default for the pact directory is provided by the jest-pact wrapper `jest-pact/dist/v3`.
 
 ```js
-import { pactWith } from 'jest-pact/v3';
-import { MatchersV3 } from '@pact-foundation/pact';
+import { pactWith } from 'jest-pact/dist/v3';
+import { MatchersV3 as Matchers } from '@pact-foundation/pact';
```

Comment:
I don't think this alias is a good idea. `Matchers` and `MatchersV3` are different and not interchangeable (unfortunately), so it makes the later code misleading.

(whether the design choice of having not-interchangeable matchers modules was a good idea is a separate discussion)

Reply:
I remove the alias